chef/automate
# Automate CDS install First pass

### :nut_and_bolt: Description: What code changed, and why?
When the install button is pressed in the UI the backend automate-cds will log that it is installing. This also does not show the "install" button for cookbook content items. 

### :chains: Related Resources

https://github.com/chef/automate/issues/3897

### :athletic_shoe: How to Build and Test the Change
1. `build components/automate-gateway && build components/automate-cds && build components/automate-ui-devproxy && start_all_services && start_automate_ui_background && ui_logs && sl | grep "automate-cds.default"`
1. Open https://a2-dev.test/cds
1. Ensure that the cookbook content item does not show the "install" button. 
1. Click one of the "install" buttons and check the logs to see that the automate-cds logged that it is installing. 
1. Success banner will be displayed in the UI also. 

### :white_check_mark: Checklist
